### ISSUE TITLE
Does reduce-motion help address 2.2.2 Pause, Stop, Hide, or does 2.2.2 require an onscreen mechanism?

### ISSUE THREAD

**Issue by jamieherrera** [w3c/wcag#4319]
2.2.2 includes in the normative text, 

> ....there is a [mechanism](https://www.w3.org/WAI/WCAG22/Understanding/pause-stop-hide#dfn-mechanism) for the user to [pause](https://www.w3.org/WAI/WCAG22/Understanding/pause-stop-hide#dfn-paused), stop, or hide it...

mechanism is defined in WCAG for 2.2.2 (and for other SC) as

> [process](https://www.w3.org/WAI/WCAG22/Understanding/pause-stop-hide#dfn-process) or technique for achieving a result
>Note 1
>The mechanism may be explicitly provided in the content, or may be [relied upon](https://www.w3.org/WAI/WCAG22/Understanding/pause-stop-hide#dfn-relied-upon) to be provided by either the platform or by [user agents](https://www.w3.org/WAI/WCAG22/Understanding/pause-stop-hide#dfn-user-agent), including [assistive technologies](https://www.w3.org/WAI/WCAG22/Understanding/pause-stop-hide#dfn-assistive-technology).
>Note 2
>The mechanism needs to meet all success criteria for the conformance level claimed.

So... if a mechanism can be provided at the platform level, could prefers-reduced-motion address some select scenarios for 2.2.2 as well as for [2.3.3 Animation from Interactions] (https://www.w3.org/WAI/WCAG22/Understanding/animation-from-interactions)?

**Comment by patrickhlauke** [comment]
x-link https://github.com/w3c/wcag/issues/4318

**Comment by detlevhfischer** [comment]
> The mechanism needs to meet all success criteria for the conformance level claimed.

Just for the record , switch to activate "Reduce motion" in iOS does not meet 1.4.11 Not-text contrast so strictly speaking in that environment the mechanism would not meet all SCs on the level claimed.

**Comment by straydogstrut** [comment]
> Related https://github.com/w3c/wcag/issues?q=is%3Aissue%20state%3Aopen%20prefers-reduced-motion - this has been discussed a few times before. I believe current thinking is that yes, it's a valid mechanism

Apologies for the necropost, but I know this has been asked before so i'm loathe to open another issue on this. However, I just wanted to clarify this conclusion.

From the links shared, the consensus isn't readily apparent to me. I appreciate issues will open/close all the time, but I made best efforts to look at any relevant ones. For example:

* [#988](https://github.com/w3c/wcag/issues/988) quite rightly imho raises the question of whether 1) the user is aware of the setting and 2) their web browser supports it. That thread doesn't conclude that it is sufficient.
* [#4318](https://github.com/w3c/wcag/issues/4318) points out that 2.3.3 links to the prefers-reduced-motion techniques, but 2.5.4 does not. This doesn't appear to have changed and has no decision regarding 2.2.2.
* [#3766](https://github.com/w3c/wcag/issues/3766) linked from the above, suggests (quite appropriately) using prefers-reduced-motion in combination with a pause button, and that the latter is what actually passes 2.2.2, implying that prefers-reduced-motion alone is not sufficient.

I did come across more recent [discussion elsewhere](https://accessibilitycraft.com/104-wcag-pause-stop-hide-prefers-reduced-motion-fallout-nuka-cola-quantum/) that comes to the conclusion that, implemented appropriately, it is technically a sufficient technique, but that it is not in spirit of WCAG due to the users being unaware of the setting /  not having the ability to enable this setting. Similar sentiments from [Hidde de Vries](https://hidde.blog/meeting-2-22-pause-stop-hide-with-prefers-reduced-motion/) a few years previous.

It might be that there's further discussion outside these tickets that has fed into the above, but is it generally accepted that prefers-reduced-motion is sufficient - if implemented correctly - but ideally an onscreen mechanism would be provided? 

(note: i'm referring only to where it makes sense to use prefers-reduced-motion.. i.e. you wouldn't use it to prevent meaningful updates)

As a real example, [this university website](https://www.dundee.ac.uk/) has an autoplaying search field animation that is presented in parallel with other content, and arguably the part that changes is not essential (it has other issues but i'm not focusing on those here). Would prefers-reduced-motion be sufficient, although not ideal here? 

I’m not trying to find a way to avoid an on screen UI option - this would be my favoured preference combined with defensive use of prefers-reduced-motion - I guess my question is about these subtle animations that, short of a site-wide option for users to disable animations, don't have an obvious place for an on screen mechanism, so would prefers-reduced-motion qualify?
